mod-source-record-storage-server/src/test/java/org/folio/services/RecordServiceTest.java

```diff
@@ -2399,4 +2564,13 @@ private void compareAdditionalInfo(TestContext context, AdditionalInfo expected,
   private void compareExternalIdsHolder(TestContext context, ExternalIdsHolder expected, ExternalIdsHolder actual) {
     context.assertEquals(expected.getInstanceId(), actual.getInstanceId());
   }
+
+  private static <T> T clone(T obj, Class<T> type) {
```

Comment:
it seems we could use the TestUtil.clone method here that we already use in this class or MarcUtil.clone method, because they are identical